rest/model/dns/zone.go

```diff
@@ -14,7 +14,10 @@ type Zone struct {
 	Pool         string   `json:"pool,omitempty"` // Deprecated
 
 	ID   string `json:"id,omitempty"`
-	Zone string `json:"zone,omitempty"`
+	Zone string `json:"zone,omitempty"` // FQDN
+	// Unique identifier for the zone within account. In most cases, it is best
+	// to also use the zone FQDN for this.
+	Name string `json:"name,omitempty"`
```

Comment:
just my 2 cents, I think I'd rather this be `ZoneName` to stay consistent with `Record`

Reply:
I'd agree, but this matches the API, where it's name on zone, and zone_name on records